classes/decision/types/Accept.php

```diff
@@ -80,7 +80,7 @@ public function getCompletedLabel(): string
 
     public function getCompletedMessage(Submission $submission): string
     {
-        return __('editor.submission.decision.accept.completedDescription', ['title' => $submission->getLocalizedFullTitle()]);
+        return __('editor.submission.decision.accept.completedDescription', ['title' => $submission?->getCurrentPublication()?->getLocalizedFullTitle(null, 'html') ?? '']);
```

Comment:
I don't think `$submission` or `$submission->getCurrentPublication()` should ever be null, right? If we don't need the null-safe operator, it's best not to use it. Likewise for `?? ''`. If these are to prevent IDE warnings about potential return values, I'd rather we solved it a different way; otherwise `?` is going to really pepper our codebase. (It's possible I'm a little out of touch on this -- recommendations welcome.)

Reply:
> I don't think `$submission` or `$submission->getCurrentPublication()` should ever be null, right? 

@asmecher I also like to think so . But the previous implementation using the `PKPSubmission::getLocalizedFullTitle` has implementation such as 
```php
$publication = $this->getCurrentPublication();
if (!$publication) {
    return '';
}
return $publication->getLocalizedFullTitle();
```
It seems that it assume that in some cases `$publication` can be null or not exist . If that is the possible case in worst case scenario, may be we should use it . 

Or is it totally impossible to be null ?

Reply:
If there was previously a case where `$publication` was checked for existence, it makes sense to carry it forward rather than trying to refactor it now. But I don't think that should be possible, unless possibly for some brief time between operations when a submission is being created.